index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>mobile version</title>
+</head>
+<body>
+  <header class="header">
+      <a>Arrow Dan</a>
+      <nav class="navbar container">
+        <img class="hamburger" src="image/Union.png" alt="navigation icon">
+      </nav>
```

Comment:
- [ ] Kindly add this image to your images folder, because it does not exist.

![image](https://user-images.githubusercontent.com/35535130/159244011-40859440-6be4-432f-89fc-144582810d77.png)

Reply:
- [ ] Please kindly make the required changes